bridges/SIMARBridge.php

```diff
@@ -0,0 +1,63 @@
+<?php
+class SIMARBridge extends BridgeAbstract {
+	const NAME = 'SIMAR';
+	const URI = 'http://www.simar-louresodivelas.pt/';
+	const DESCRIPTION = 'Verificar estado da rede SIMAR';
+	const MAINTAINER = 'somini';
+	const PARAMETERS = array(
+		'Público' => array(
+			'interventions' => array(
+				'type' => 'checkbox',
+				'name' => 'Incluir Intervenções?',
+				'defaultValue' => 'checked',
+			)
+		)
+	);
+
+	public function collectData() {
+		$html = getSimpleHTMLDOM(self::getURI())
+			or returnServerError('Could not load content');
+		$e_home = $html->find('#home', 0)
+			or returnServerError('Invalid site structure');
+
+		foreach($e_home->find('span') as $element) {
+			$item = array();
+
+			$item['title'] = 'Rotura: ' . $element->plaintext;
+			$item['content'] = $element->innertext;
+			$item['uri'] = 'urn:sha1:' . hash('sha1', $item['content']);
```

Comment:
This will not produce a valid URI (at least it is not one pointing to a valid destination), which makes it pretty useless in my opinion. If your goal is to make the item unique, consider using the recently added ['uid'](https://github.com/RSS-Bridge/rss-bridge/wiki/The-collectData-function#item-parameters) element like so:

```suggestion
			$item['uid'] = 'urn:sha1:' . hash('sha1', $item['content']);
```

Note: You can provide it any string and it will hash it automatically.

Reply:
Makes sense, I really want the feed reader to consider them separate elements.